### DISCUSSION TITLE
Direct Messages lists only the Active Users and not the entire User List

### DISCUSSION THREAD

**Discussion by zaheer-srushty** [The-Commit-Company/raven#694] [General]
The Direct Messages window shows only the Active Users and not the entire User list. And due to issue #691 , we cannot send a DM to an inactive user. Please let me know if this is by design or a bug.

Also, it would be great if the unread messages/channels push them to the top of each category.

Kudos to the team for the work you have done till date for bringing Raven to this maturity. I hope I could be of some help in the days to come.

Please let me know how I could help you with this.

**Comment by nikkothari22** [comment]
Hey @zaheer-srushty, #691 is fixed now, and we were in the middle of a big performance upgrade. Not that it's complete, we'll add a section on the sidebar for unread channels/DMs (#773)

About the sidebar not showing all users - that's by design. It would consume a lot of unnecessary space if there are lots of users.
